sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dmac_training.py

```diff
@@ -25,10 +25,7 @@ class TestDMACTraining(FormRecognizerTest):
     @DocumentModelAdministrationClientPreparer()
     @recorded_by_proxy
     def test_build_model_polling_interval(self, client, formrecognizer_storage_container_sas_url, **kwargs):
-        # this can be reverted to set_bodiless_matcher() after tests are re-recorded and don't contain these headers
-        set_custom_default_matcher(
-            compare_bodies=False, excluded_headers="Authorization,Content-Length,x-ms-client-request-id,x-ms-request-id"
-        )
+        set_bodiless_matcher()
```

Comment:
These tests do need bodiless matcher because of the random model ids being generated